File: app/src/main/java/com/petal/browser/fragment/Fragment_settings_Delete.java
package com.petal.browser.fragment;


import android.os.Bundle;

import com.petal.browser.R;
import com.petal.browser.preferences.BasePreferenceFragment;

public class Fragment_settings_Delete extends BasePreferenceFragment {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preference_delete, rootKey);
    }
}

File: app/src/main/java/com/petal/browser/fragment/Fragment_settings_Delete.kt
package com.petal.browser.fragment

import android.os.Bundle
import com.petal.browser.R
import com.petal.browser.preferences.BasePreferenceFragment

class Fragment_settings_Delete : BasePreferenceFragment() {

    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preference_delete, rootKey)
    }
}
